s3update.go

```diff
@@ -114,7 +116,7 @@ func runAutoUpdate(u Updater) error {
 		if err != nil {
 			return err
 		}
-		defer resp.Body.Close()
+		remoteFileSize = *resp.ContentLength
```

Comment:
Maybe we should still keep the call to `resp.body.Close` to guarantee no matter each path we are taking this is properly closed
```suggestion
              defer resp.Body.Close()
		remoteFileSize = *resp.ContentLength
```